itest/e2e_test.go

```diff
@@ -98,14 +98,11 @@ func TestDoubleSigning(t *testing.T) {
 		Height: finalizedBlocks[0].Height,
 		Hash:   datagen.GenRandomByteArray(r, 32),
 	}
-	_, _, err = fpIns.TestSubmitFinalitySignatureAndExtractPrivKey(b)
-	require.Error(t, err)
-
-	// todo(lazar): implement UnsafeSignEOTS, containing logic without double sign protection
-	//require.NoError(t, err)
-	//require.NotNil(t, extractedKey)
-	//localKey := tm.GetFpPrivKey(t, fpIns.GetBtcPkBIP340().MustMarshal())
-	//require.True(t, localKey.Key.Equals(&extractedKey.Key) || localKey.Key.Negate().Equals(&extractedKey.Key))
+	_, extractedKey, err = fpIns.TestSubmitFinalitySignatureAndExtractPrivKey(b)
```

Comment:
I was thinking that maybe we should keep an option to use `SignEOTS` within `TestSubmitFinalitySignatureAndExtractPrivKey` other than only the unsafe one so that we can test with the slashing protection works